requirements/README.md

```diff
@@ -0,0 +1,26 @@
+
+# Requirements Engineering
+To be able to quantify the quality of community contributions, it needs to be known what the requirements are for the contributions in question. Then the contribution needs to be tested against this requirement in some way. If the requirement changes, the contribution needs to be changed as well to reflect this. This process needs to be structured to prevent it from getting too complex.
```

Comment:
First plural `it needs to be known what the requirements` then singular `Then the contribution needs to be tested against this requirement`. Furthermore I think this could be written down much shorter.

I also feel the definition of requirements engineering should come before explaining why it is necessary.

Perhaps these two paragraphs could be rewritten to
> The field of requirements engineering is a part of systems and software engineering, and deals with structuring requirements. Requirements are necessary to be able to quantify the quality of community contributions. Requirements engineering involves systems modeling and the inception, identification etc.